_html/scss/modules/_user-form.scss

```diff
@@ -1,31 +1,32 @@
-.user-form {
-    margin-top: 10rem;
-
-    .form-item-wrapper-content {
-        display: inline-block;
-        width: calc(50% - 1.2rem);
-        vertical-align: top;
-
-        &.form-item-pass,
-        .form-item-field-gender>label {
-            width: 100%;
-        }
-    }
-
-    .form-item-wrapper {
-        margin-bottom: 2rem;
+label {
+    width: 100%;
```

Comment:
Why do you include this general rule here? I think there is a file called `elements` where we set super general rules on tags. I think this would go better there.